### PR TITLE
Added an X to dismiss the invalid markets notice for user session

### LINKED ISSUE #4159: Alert bar from filters settings at the top of the market list pg needs an (x) to be closed added
Need to add an (x) in the top right of this bar...so user can dismiss it if they choose

------------------------------------------
tasks:
1. add X so user can dismiss the alert
2. store in state (on loginAccount on new `settings` property) that the user dismiss this alert.
3. check loginAccount state before showing this alert

### PR DESCRIPTION
**Pull request by bconfortin**
Closes https://github.com/AugurProject/augur/issues/4159

### CODE REVIEW

File: packages/augur-ui/src/modules/markets-list/actions/update-login-account-settings.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import {Action} from 'redux';
+import {LoginAccountSettings} from "modules/types";
+
+export const updateLoginAccountSettings = (
+  settings: LoginAccountSettings = {},
+) => (dispatch: ThunkDispatch<void, any, Action>, getState: () => AppState) => {
+  const { loginAccount } = getState();
+  try {
+    console.log('settings', settings);
+    console.log('loginAccount', loginAccount);
```
Comment (bthaile) [inline packages/augur-ui/src/modules/markets-list/actions/update-login-account-settings.ts]: Prob dont need these console logs

**Review by bthaile — COMMENTED** [review]
Code looks good.

File: packages/augur-ui/src/modules/markets-list/actions/update-login-account-settings.ts (hunk 1 above)
Comment (bconfortin) [reply packages/augur-ui/src/modules/markets-list/actions/update-login-account-settings.ts]: Oops

**Review by bthaile — APPROVED** [review]
Looks good. I like the typing. 

You check mobile?

**Comment by bthaile** [comment]
Check mobile on emulator

**Comment by bconfortin** [comment]
> Looks good. I like the typing.
> 
> You check mobile?

@bthaile I have tested it on Chrome only. I still have to figure out how to emulate on Linux. Will probably do that tomorrow, in the morning.

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 💪

 This link will always point to this PR's latest changes: https://peek.run/exoh050

**Pull request merged**